### PR TITLE
update node version to 16 for ember-beta

### PR DESCRIPTION
**Pull request by mansona**
Since ember-cli@5 requires a minimum of Node@16 I have updated that in our volta pin 👍

### CODE REVIEW

**Comment by mansona** [comment]
So I think I need a bit of help to continue with this one. For some reason the git diff isn't applying in the CI environment but it is applying properly locally 🤔 

I thought it might be an issue with a difference between my git version locally and the one in the github actions but I tried updating both and it didn't seem to do anything 😞 

One thing that I noticed is that the git diff looked a bit odd locally, it had some extra stuff that looked like it shouldn't be there.

```diff
@@ -9,2 +9,3 @@ module.exports = function (environment) {
     EmberENV: {
+      RAISE_ON_DEPRECATION: true,
       FEATURES: {
```

The `module.exports = function (environment) {` seems out of place in that patch 🤔 and sure enough if I left that it would fail locally. But I think I tried leaving it in and it still failied on the CI 😞 

Any help here would be much appreciated 🎉

File: src/markdown/tutorial/part-1/01-orientation.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ del package.json
 @@ -9,2 +9,3 @@
      EmberENV: {
 +      RAISE_ON_DEPRECATION: true,
-       EXTEND_PROTOTYPES: false,
+       FEATURES: {
```
Comment (kategengler) [inline src/markdown/tutorial/part-1/01-orientation.md:93]: I'm not sure where this change is coming from -- afaict new apps on v5.0.0 still have `EXTEND_PROTOTYPES: false,` as the next line here.

Comment (mansona) [reply src/markdown/tutorial/part-1/01-orientation.md:93]: strange 🤔 I must have a local dependency issue

**Comment by kategengler** [comment]
#227 pulled in your node update -- did you want to land any of the other changes in this PR or should it be closed?

**Pull request closed without merging**